### PR TITLE
Don't instrument libFuzzer when building atheris.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Sorry, I accidentally cancelled some of the CI jobs here to debug why CIFuzz was going so slowly.

**Comment by inferno-chromium** [comment]
> Sorry, I accidentally cancelled some of the CI jobs here to debug why CIFuzz was going so slowly.

No worries.

File: infra/base-images/base-builder/Dockerfile [hunk 1]
```diff
@@ -59,7 +59,14 @@ RUN export PYTHON_DEPS="\
     apt-get remove -y $PYTHON_DEPS  # https://github.com/google/oss-fuzz/issues/3888
 
 # Install latest atheris for python fuzzing, pyinstaller for fuzzer packaging.
-RUN unset CFLAGS CXXFLAGS && pip3 install -v atheris pyinstaller==4.1
+RUN unset CFLAGS CXXFLAGS && \
```
Comment (jonathanmetzman) [inline infra/base-images/base-builder/Dockerfile:62]: Does atheris use a different libFuzzer version than the rest of our fuzz targets?

**Comment by inferno-chromium** [comment]
Not needed and incorrect.

**Pull request closed without merging**